services-manager/src/main/resources/application-services.yaml

```diff
@@ -37,46 +37,46 @@ manager:
       jar: audit-service/target/audit-service-${app.version}-exec.jar
       paths: *libraries
       profiles:
-        - eureka
```

Comment:
should the list be indented or not?

Reply:
Not going to lie, i just hit the tidy code button. IntelliJ seems to want to standardise it to w/o the indent

Reply:
I think it reads better with the indent. although not strictly required

Reply:
Depends whether or not we should be using IntelliJ's automatic code tidying... Up to you @ac74475